.github/workflows/python-package.yml

```diff
@@ -10,6 +10,7 @@ on:
   pull_request:
     branches:
      - master
+  workflow_dispatch:
```

Comment:
Does this work? It looks like it would do nothing?

Reply:
For reference: [https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow)